test/e2e/storage/testsuites/testdriver.go

```diff
@@ -147,6 +147,7 @@ const (
 	CapRWX                 Capability = "RWX"                 // support ReadWriteMany access modes
 	CapControllerExpansion Capability = "controllerExpansion" // support volume expansion for controller
 	CapNodeExpansion       Capability = "nodeExpansion"       // support volume expansion for node
+	CapVolumeLimits                   = "volumeLimits"        // support volume limits (applicable only to CSI drivers)
```

Comment:
do we want to also support in-tree so that we can test migration?

Reply:
yeah, volume limits are also supported in-tree so testing both that and the migration from in-tree would be useful.

Reply:
In-tree drivers have their limits somewhere else. Is there a way how to detect a testdriver is CSI or not?

Reply:
Found it, `driverInfo.InTreePluginName`.